tests/__init__.py

```diff
@@ -420,68 +428,118 @@ def start_xdp(self):
         Start the xdg-desktop-portal process
         """
 
+        portal_dir = (
+            Path(os.getenv("G_TEST_BUILDDIR") or "tests")
```

Comment:
```suggestion
            Path(os.getenv("G_TEST_BUILDDIR", "tests"))
```
though admittedly it does the same thing...